src/app/effects/api-search.effects.ts

```diff
@@ -118,6 +118,7 @@ export class ApiSearchEffects {
 				),
 				withLatestFrom(this.store$),
 				map(([action, state]) => {
+					this.pushService.postData(action['payload']);
```

Comment:
It is otiose to push data to the backend when you are actually getting the data from the backend. It would already contain all the statuses in this case.

The push servlet is supposed to be used when we are fetching Tweets using Twitter API they may be new for the backend.

Reply:
Yes, I agree it would be more helpful once we complete the twitter api support. But till then it could work I guess, after that we can just push the mapped data from twitter api i.e. the data extracted from twitter api mapped in format of api.loklak. I have made all other changes, and I will surely write the tests in further patches. Thanks @singhpratyush